scalableminds/webknossos
# Remove Bootstrap

I removed Bootstrap and weeded out some unused CSS classes / styles.

### Steps to test:
- Check the layout of all admin views
- Check the layout of the tracing view (public, skeleton, volume)

### Issues:
- fixes #2009 

### Remaining issues
- [x] [Admin.Tasks] The grid is messed up. User Selection is on its own row.
![image](https://user-images.githubusercontent.com/1105056/33385581-3ca7c884-d529-11e7-899b-01cbab3f40d6.png)
- [x] [Skeleton/Volume Tracing View] The action buttons at the top are slightly offset
![image](https://user-images.githubusercontent.com/1105056/33385653-6760e9a2-d529-11e7-9267-c652f8065dcd.png)


------
